core/api.go

```diff
@@ -148,6 +152,15 @@ type PushConsumer interface {
 	Subscribe(topic, expression string, consumeFunc func(msg *MessageExt) ConsumeStatus) error
 }
 
+// PullConsumerConfig the configuration for the pull consumer
+type PullConsumerConfig struct {
+	ClientConfig
+}
+
+func (config *PullConsumerConfig) String() string {
+	return "PushConsumerConfig=[" + config.ClientConfig.string() + "]"
```

Comment:
config.ClientConfig.`string()` is the typo of config.ClientConfig.`String()` ?

Reply:
no type, `ClientConfig` is not a export struct in the past, and `string()` of it is a helper method for `String()` of `Producer/PushConsumer/PullConsumerConfig`. in order to avoid ambiguity, i will change it to `String` from `string`

Reply:
fixed